boilerplates/react-union-boilerplate-monorepo/packages/union-app-sample-app/src/index.js

```diff
@@ -5,6 +5,7 @@ import { AppContainer } from 'react-hot-loader';
 
 import Root from './components/Root';
 
+// TODO: refactor to use the new `hot(module)(Root)` API
```

Comment:
remove TODO.. it does not belong to the boilerplate

Reply:
This TODO is related to merging the entire `ssr` branch. I will resolve this TODO before merging.